src/style.css

```diff
@@ -1 +1,25 @@
 /* styles go here */
+input,
+textarea {
```

Comment:
Use classes for styling, not tags
![image](https://user-images.githubusercontent.com/99403198/233616084-28693399-51f9-4763-8f9b-a500842a860c.png)